docs/faq-and-troubleshooting/common-questions/fifa-and-eafc-not-launching.md

```diff
@@ -0,0 +1,10 @@
+---
+description: FIFA/EAFC not launching
+icon: simple/fifa
+---
+
+# :simple-fifa: Why are FIFA and EAFC not launching?
+
+Both FIFA and EAFC require Windows Defender to be enabled and running.
```

Comment:
Other pages usually mark the main point in bold; it would also be best to do that here. By the way, would you happen to have any examples of this issue?